src/amo/components/AddonMoreInfo.js

```diff
@@ -32,7 +32,9 @@ export class AddonMoreInfoBase extends React.Component {
             {addon.current_version.version}
           </dd>
           <dt>{i18n.gettext('Last updated')}</dt>
-          <dd>{addon.last_updated}</dd>
+          <dd>
+            {i18n.moment(addon.last_updated).fromNow()} ({i18n.moment(addon.last_updated).format('MMMM Do YYYY')})
```

Comment:
Should this be localised? I'm not sure if parens translate well and some languages might want something like "1 year ago on December 1, 2015"